doc/employees.md

```diff
@@ -1 +1,1220 @@
 # Employees Module
+Module | Description 
+--- | --- | --- | ---
+[Add](#add) | POST | Add New Data Employee
+[View](#view) | GET | View Data Employee 
+[Edit](#edit) | PUT | Edit Data Employee 
+[Delete](#delete) | DELETE | Delete Data Employee
+
+## <a name="add"></a>Add
+
+### Endpoint 
+POST /employees
+
+### Database
+![](./employee.png)
+
+for add, you need get company_id from companies tables and branch.id from branches tables
+
+### Headers
+Key | Value 
+--- | ---
+Content-Type | application/json
+Accept | application/json
+
```

Comment:
Is there a filter from the currently logged in user?
For example : 
If the login is a user of A company, the employee created is automatically the employee of A company.
If there is a filter that is taken from a logged in user, where do you get the information on who is currently logged in?
May be you can selected from the following 2 options:
1. Add the user_id information in the header, to confirm that who is currently logged in is the user_id as sent in the header. Other required user fields (ex company_id, branch_id, etc) can be obtained from the user_id already in the header.
2. Add the user login information in the json body, so that the json request body format is:
```
{
    user_login :{
        "id": 1,
        "company_id": 11,
        "branch_id": 7
   },
    "id": "2",
    "employee_code": "20200301",
    "employee_name": "ferry",
    "employee_address": "Jl.Mangga Besar XIII"
}
```

I personally prefer the first option.

Please confirm to Pak @wahablie

Reply:
ok, thankyou sir @jacky-htg

Reply:
using email in header to get user login information